### ISSUE TITLE
Feature Request: DNS Recursor

### ISSUE THREAD

**Issue by hyperxpro** [dnsjava/dnsjava#141]
Since we already have an excellent DNS implementation, We should implement DNS Recursor in `dnsjava`.

**Comment by hyperxpro** [comment]
@ibauersachs Should I start PR with a very basic implementation recursor?

**Comment by ibauersachs** [comment]
Yes if you want to. But this isn't a trivial job and there need to be plenty of tests for it to make sure all relevant RFCs are adhered to.

**Comment by hyperxpro** [comment]
I did found some broken implementations on the Internet but none of them work correctly. Do you've any timeline or updates on this?

**Comment by ibauersachs** [comment]
Not sure what you mean with having a timeline, but no, there isn't one. And for the avoidance of doubt, I'm not working on a recursive resolver myself nor will I do so in the future.

**Comment by hyperxpro** [comment]
Ah, that's fine then.
